### PR TITLE
added the NetbeansIDE configuration page

### PR DESCRIPTION
**Pull request by jeremp**
I tried to use my favorite IDE (Netbeans) and it seems to work well, so i added it to the documentation.

### CODE REVIEW

File: pages/configuring_ide_netbeans.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: Configuring your IDE
+permalink: /configuring-ide-netbeans/
+redirect_from:
+  - /configuring_ide.html
```
Comment (jdubois) [inline pages/configuring_ide_netbeans.md]: should be: /configuring_ide_netbeans.html

File: pages/configuring_ide_netbeans.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+Importing your JHipster application in Netbeans is straightforward.
+
+## 1. Open your project
+
+- Select File -> Open Project
+- Browse and select your project's folder
+
+![Open]({{ site.url }}/images/configuring_ide_netbeans_1.png)
+
+**That's All** : To run your project, just do a right-click on the Application class and select "Run File".
```
Comment (jdubois) [inline pages/configuring_ide_netbeans.md]: Maybe you could add that the "Application" class is at the root of the package hierarchy

**Comment by jdubois** [comment]
Thanks! I've added a few comments, but this should be merged soon

File: pages/configuring_ide_netbeans.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: Configuring your IDE
+permalink: /configuring_ide-netbeans/
+redirect_from:
+  - /configuring_ide.html
```
Comment (jdubois) [inline pages/configuring_ide_netbeans.md]: This should be /configuring_ide_netbeans.html

File: pages/configuring_ide_netbeans.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: Configuring your IDE
+permalink: /configuring_ide-netbeans/
```
Comment (jdubois) [inline pages/configuring_ide_netbeans.md]: this should be /configuring-ide-netbeans/
(you need the dash for Google)

Comment (jeremp) [inline pages/configuring_ide_netbeans.md]: Sorry i missed that one, i correct it now

File: pages/configuring_ide_netbeans.md (hunk 1 above)
Comment (deepu105) [inline pages/configuring_ide_netbeans.md]: Actually redirect from is not required for new pages

Comment (jeremp) [inline pages/configuring_ide_netbeans.md]: should i remove it ?

Comment (jdubois) [inline pages/configuring_ide_netbeans.md]: I'll add it for consistency... I'm going to merge this, don't worry that's my job now :-)

**Comment by jdubois** [comment]
It's up and running at http://jhipster.github.io/configuring-ide-netbeans/

Thanks @jeremp !!

**Comment by jeremp** [comment]
It was a pleasure to contribute.

**Pull request merged**